plugins/flytekit-kf-pytorch/flytekitplugins/kfpytorch/task.py

```diff
@@ -123,13 +123,15 @@ class Elastic(object):
         start_method (str): Multiprocessing start method to use when creating workers.
         monitor_interval (int): Interval, in seconds, to monitor the state of workers.
         max_restarts (int): Maximum number of worker group restarts before failing.
+        rdzv_configs (Dict[str, Any]): Additional rendezvous configs to pass to torch elastic.
```

Comment:
```suggestion
        rdzv_configs (Dict[str, Any]): Additional rendezvous configs to pass to torch elastic, e.g. `{"timeout": 1200}`.
            See `torch.distributed.launcher.api.LaunchConfig`.
```

@Nan2018 is `{"timeout": 1200}` (in seconds) the correct usage?